credentialz/credentialz.proto

```diff
@@ -592,7 +597,7 @@ enum KeyGen {
   KEY_GEN_SSH_KEY_UNSPECIFIED = 0;
   KEY_GEN_SSH_KEY_TYPE_RSA_2048 = 1;
   KEY_GEN_SSH_KEY_TYPE_ECDSA_P_256 = 2;
-  KEY_GEN_SSH_KEY_TYPE_ECDSA_P_521 = 3;
```

Comment:
The name of the curve is `P-521` so `KEY_GEN_SSH_KEY_TYPE_ECDSA_P_521` is valid.

Reply:
The latest proto file still has `KEY_GEN_SSH_KEY_TYPE_ECDSA_P_512` instead of `KEY_GEN_SSH_KEY_TYPE_ECDSA_P_521`